v2 npm scripts are broken

All scripts are failing with:

```
module.js:341
    throw err;
    ^

Error: Cannot find module '../webpack.config.babel'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/sgillespie/dev/availity-uikit/dev/start.js:9:23)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/sgillespie/dev/availity-uikit/dev/index.js:7:10)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
```

--- a/dev/docs.js
+++ b/dev/docs.js
@@ -4,7 +4,7 @@ const webpack = require('webpack');
 
 const Logger = require('./logger');
 const metalsmith = require('./metalsmith');
-const wepackConfig = require('../webpack.config.babel');
+const wepackConfig = require('../webpack.config');
 
 function bundle() {
 

--- a/dev/start.js
+++ b/dev/start.js
@@ -6,7 +6,7 @@ const webpack = require('webpack');
 const metalsmith = require('./metalsmith');
 const Logger = require('./logger');
 const watch = require('./watch');
-const webpackConfig = require('../webpack.config.babel');
+const webpackConfig = require('../webpack.config');
 
 function serv() {
 

